yarn.lock

```diff
@@ -199,6 +203,12 @@
   version "1.4.31"
   resolved "https://registry.yarnpkg.com/@types/socket.io-client/-/socket.io-client-1.4.31.tgz#86b8986409384a9ee1a74993a70822c36114dc79"
 
+"@types/urijs@^1.15.37":
+  version "1.15.37"
+  resolved "https://registry.yarnpkg.com/@types/urijs/-/urijs-1.15.37.tgz#42cd9d5af585053e9af5d009d77bfeb6b0b4cfe6"
+  dependencies:
+    "@types/jquery" "*"
```

Comment:
What on earth is this doing here.  I checked... urijs definitely doesn't depend on jQuery.  And if it did, I would be removing it tomorrow morning haha.

Reply:
Looks like `urijs` has a jQuery-plugin version.